src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.OpenFileTracker.cs

```diff
@@ -354,11 +360,17 @@ private void TryClosingDocumentsForCookie(uint cookie)
                             {
                                 w.OnDocumentClosed(documentId, new FileTextLoader(moniker, defaultEncoding: null));
                             }
-                            else
+                            else if (w.CurrentSolution.ContainsAdditionalDocument(documentId))
                             {
-                                Debug.Assert(w.CurrentSolution.ContainsAdditionalDocument(documentId));
                                 w.OnAdditionalDocumentClosed(documentId, new FileTextLoader(moniker, defaultEncoding: null));
                             }
+                            else
+                            {
+                                // TODO: implement the ability for analyze config documents to be opened against
+                                // a live text editor. This is tracked by
+                                // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/750120
+                                Debug.Assert(w.CurrentSolution.ContainsAnalyzerConfigDocument(documentId));
```

Comment:
📝 If `OnAnalyzerConfigDocumentOpened` is created as a stub, so should `OnAnalyzerConfigDocumentClosed`

Reply:
(see earlier comment: I don't think adding a stub is wise, given it would be a public API)